plugins/pipelines/create.js

```diff
@@ -31,15 +31,15 @@ module.exports = (server, options) => ({
                 }, next),
                 (permissions, next) => {
                     if (!permissions.admin) {
-                        return reply(boom.unauthorized(`User ${username} `
+                        return next(boom.unauthorized(`User ${username} `
                             + 'is not an admin of this repo'));
                     }
 
                     return Pipeline.get(pipelineId, next);
                 },
                 (data, next) => {
```

Comment:
Hmm... this is not the scope of this PR. I'm just changing the `reply` to `next`. But since this is a simple change, I can do that.